src/components/functional_module/src/plugin_manager.cc

```diff
@@ -43,6 +43,38 @@ namespace functional_modules {
 
 CREATE_LOGGERPTR_GLOBAL(logger_, "PluginManager")
 
+namespace {
+const std::string ExtractMethodName(application_manager::MessagePtr msg) {
```

Comment:
@dev-gh dcheck msg

Reply:
@dev-gh nobody know where else this method will be called and what data will be passed to it

Reply:
@LuxoftAKutsan 
Done with dca9f08